matrix.go

```diff
@@ -348,48 +349,107 @@ type NewRoomBackfillRequest struct {
 	Chats []*imessage.ChatInfo `json:"chats"`
 }
 
-type NewRoomForBackfill struct {
-	RoomID                   id.RoomID
-	EarliestBridgedTimestamp int64
-}
-
-type NewRoomBackfillResponse map[string]NewRoomForBackfill
+var createRoomsForBackfillError error = nil
```

Comment:
This should probably be in `WebsocketCommandHandler` to look slightly less hacky (although even then it's very hacky)